### DISCUSSION TITLE
Prisma

### DISCUSSION THREAD

**Discussion by PedroFnseca** [Buscafe/Backend#9] [Q&A] [answered]
O que é este prisma?

**Comment by Igorcbraz** [comment, marked as answer]
Prisma desbloqueia um novo nível de experiência de desenvolvedor ao trabalhar com bancos de dados graças ao seu modelo de dados intuitivo, migrações automatizadas, segurança de tipo & conclusão automática.

**Reply by PedroFnseca** [reply]
Poxa não sabia!
